utils/getScenarioContent.ts

```diff
@@ -243,8 +244,17 @@ export function byeScenario(claimData: Claim): ScenarioType | null {
         return ScenarioType.Scenario8
       case 'DUA':
         return ScenarioType.Scenario9
+
+      // BYE scenarios with the Program Type 'FED-ED' are treated differently depending
+      // on whether the benefitYearEndDate is before or on/after 2020-05-10.
       case 'FED-ED Extension':
-        return ScenarioType.Scenario12
+        if (benefitYearEndDate) {
+          if (isFirstDateBefore(benefitYearEndDate, '2020-05-10T00:00:00')) {
+            return ScenarioType.Scenario10
+          } else {
+            return ScenarioType.Scenario12
+          }
+        }
```

Comment:
So in the (badly formatted API response) case that we have `isBYE` and `FED-ED` `programType`, but `benefitYearEndDate` is undefined - we will fall through to Scenario5 as we've written this. (we'll fall thru to return null, meaning 299 will be false, falling us through to scenario5 on line 305)

Do we think that's the best plan, or should we have one of 10/12 be the catch?

Reply:
@lomky Yeah, I made that decision intentionally, but maybe tagging in @dianagriffin is the right next step here.

Reply:
@lomky I have updated the code. I updated this PR description to include one more test case.